doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -21,12 +21,189 @@ msgstr ""
 msgid "News"
 msgstr "お知らせ"
 
-msgid "Release 7.0.9 - 2017-11-29"
-msgstr "7.0.9リリース - 2017-11-29"
+msgid "Release 7.1.0 - 2017-12-29"
+msgstr "7.1.0リリース - 2017-12-29"
 
 msgid "Improvements"
 msgstr "改良"
 
+msgid ""
+"[:doc:`/reference/commands/load`] Improved the ``load``'s query-log format. "
+"Added detail below items in the ``load``'s query-log."
+msgstr ""
+"[:doc:`/reference/commands/load`] ``load`` コマンドのクエリーログのフォーマッ"
+"トを改良しました。``load`` コマンドのクエリーログに以下の項目が追加されます。"
+
+msgid "outputs number of loaded records."
+msgstr "ロードしたレコード数を表示します。"
+
+msgid "outputs number of error records and columns."
+msgstr "ロードがエラーになったレコードとカラムの数を表示します。"
+
+msgid "outputs number of total records."
+msgstr "ロード後の総レコード数を表示します。"
+
+msgid ""
+"[:doc:`/reference/commands/logical_count`] Improved the ``logical_count``'s "
+"query-log format. Added detail below items in the ``logical_count``'s query-"
+"log."
+msgstr ""
+"[:doc:`/reference/commands/logical_count`] ``logical_count`` コマンドの"
+"クエリーログのフォーマットを改良しました。 ``logical_count`` コマンドの"
+"クエリーログに以下の項目が追加されます。"
+
+msgid "outputs number of count."
+msgstr "マッチしたレコード数を表示します。"
+
+msgid ""
+"[:doc:`/reference/commands/logical_select`] Improve the ``logical_select``'s "
+"query-log format. Added detail below items in the ``logical_select``'s query-"
+"log."
+msgstr ""
+"[:doc:`/reference/commands/logical_select`] ``logical_select`` コマンドのクエ"
+"リーログのフォーマットを改良しました。 ``logical_select`` コマンドのクエリー"
+"ログに以下の項目が追加されます。"
+
+msgid "log N outputs."
+msgstr "マッチしたレコード数を表示します。"
+
+msgid "outputs plain drilldown."
+msgstr "ドリルダウンした数を表示します。"
+
+msgid "outputs labeled drilldown."
+msgstr "ラベルつきのドリルダウンをした数を表示します。"
+
+msgid "outputs selected in each shard."
+msgstr "それぞれのシャードで検索にマッチした数を表示します。"
+
+msgid "use \"[...]\" for target information."
+msgstr "検索対象のシャードの情報を\"[...]\"を使って表示します。"
+
+msgid ""
+"[:doc:`/reference/commands/delete`] Improved the ``delete``'s query-log "
+"format. Added detail below items in the ``delete``'s query-log."
+msgstr ""
+"[:doc:`/reference/commands/delete`] ``delete`` コマンドのクエリーログのフォー"
+"マットを改良しました。 ``delete`` コマンドのクエリーログに以下の項目が追加さ"
+"れます。"
+
+msgid "outputs number of deleted and error records."
+msgstr "削除したレコード数と削除に失敗したレコード数を表示します。"
+
+msgid "outputs number of rest number of records."
+msgstr "削除後に残ったレコード数を表示します。"
+
+msgid "[groonga] Ensured stopping by C-c."
+msgstr ""
+"[groonga] groonga -s で実行したサーバーを C-C で確実に停止出来るようにしまし"
```

Comment:
`C-c`

Reply:
`groonga -s` should be:

```rst
``groonga -s``
```

Reply:
By the way, why does the information exist only in Japanese documentation?
If it's an important information, it should also exist in English documentation.
If it's not an important information, it should not exist in Japanese documentation.